src/mono/sample/wasi/native/local.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int ManagedFunc(int number);
+
+void UnmanagedFunc()
```

Comment:
let's rename this to `MyImport` so that it's not confusing the reader of C# side

Reply:
I did the mismatched names on purpose to demonstrate what setting EntryPoint does in both cases